includes/discovery/sensors/state/helios.inc.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
Looks like you could move this into yaml: https://docs.librenms.org/#Developing/Sensor-State-Support/

Also needs test units please: https://docs.librenms.org/#Developing/os/Test-Units/#example-workflow

And finally, can you sign the CLA :)